packages/starlight/translations/ar.json

```
@@ -0,0 +1,22 @@
+{
+  "skipLink.label": "Skip to content",
```

Looks like this was not translated. What do you think of this?

```suggestion
  "skipLink.label": "انتقل إلى المحتوى",
```